specreduce/tracing.py

```diff
@@ -92,10 +109,12 @@ class FlatTrace(Trace):
     trace_pos : float
         Position of the trace
     """
-    trace_pos: float
+    _trace_pos: (float, np.ndarray) = field(repr=False)
```

Comment:
Can you help me understand when/why this `FlatTrace` argument would take an array?